--- Preface.v.orig	2020-11-03 13:22:07.226270600 +0900
+++ Preface.v	2020-11-03 19:03:09.103049200 +0900
@@ -146,14 +146,14 @@
 (** If you want to refer to this volume in your own writing, please
     do so as follows:
 
-   @book            {$FIRSTAUTHOR:SF$VOLUMENUMBER,
-   author       =   {$AUTHORS},
-   title        =   "$VOLUMENAME",
+   @book            {Pierce:SF2,
+   author       =   {Benjamin C. Pierce, Arthur Azevedo de Amorim, Chris Casinghino, Marco Gaboardi, Michael Greenberg, Cătălin Hriţcu, Vilhelm Sjöberg, Andrew Tolmach, Brent Yorgey},
+   title        =   "Programming Language Foundations",
    series       =   "Software Foundations",
-   volume       =   "$VOLUMENUMBER",
-   year         =   "$VOLUMEYEAR",
+   volume       =   "2",
+   year         =   "2020",
    publisher    =   "Electronic textbook",
-   note         =   {Version $VERSION, \URLhttp://softwarefoundations.cis.upenn.edu },
+   note         =   {Version 5.8, \URLhttp://softwarefoundations.cis.upenn.edu },
    }
 *)
 
