--- Typeclasses.v
+++ Typeclasses.v
@@ -1154,7 +1154,7 @@
     [Equivalence] (and the associated classes [Reflexive],
     [Transitive], etc.) and [Proper].  They are described in the
     second half of _A Gentle Introduction to Type Classes and
-    Relations in Coq_, by Castéran and Sozeau.
+    Relations in Coq_, by Castéran and Sozeau.
     https://www.labri.fr/perso/casteran/CoqArt/TypeClassesTut/typeclassestut.pdf.
 
     A much larger collection of typeclasses for formalizing
